ros2launch/ros2launch/command/launch.py

```diff
@@ -103,6 +107,26 @@ def add_arguments(self, parser, cli_name):
             help="Arguments to the launch file; '<name>:=<value>' (for duplicates, last one wins)")
         arg.completer = SuppressCompleterWorkaround()
 
+        sec_args = parser.add_argument_group(title='Security', description='Security related arguments')
+        arg = sec_args.add_argument(
+            '--secure',
+            metavar='keystore',
+            nargs='?',
+            const='',
+            help=('Launch node with encryption using specified keystore dir.'
+                  'Will generate keys if necessary.'),
```

Comment:
This help doc string is a little misleading I believe. 1. security is not only about encryption. 2. The whole launch file's content is affected by this option, not a single 'node'.
```suggestion
            help=('Launch using ROS 2 security features using specified keystore dir (optional).'
                  'Will set up a keystore if none specified.'),
```
This might still be wrong/incomplete tho.

Reply:
Currently this only enables encryption, don't want to oversell the functionality since permissions aren't factored in yet

Reply:
Alright fair enough, my second point is still valid tho.

Reply:
69cf810